avocado/plugins/runner_nrunner.py

```diff
@@ -254,7 +254,11 @@ def run_suite(self, job, test_suite):
         spawner_name = test_suite.config.get('nrunner.spawner')
         spawner = SpawnerDispatcher(test_suite.config)[spawner_name].obj
         max_running = test_suite.config.get('nrunner.max_parallel_tasks')
-        workers = [Worker(tsm, spawner, max_running=max_running).run()
+        timeout= test_suite.config.get('runner.timeout.process_alive')
```

Comment:
I understand what you did here reading your commit message and the code, but I confess I got confused the first time I looked at the code without reading the description.

The `runner.timeout.process_alive` is a particular parameter used at https://github.com/avocado-framework/avocado/blob/master/avocado/core/runner.py#L201 related to the test processes.

Having a new parameter starting with `nrunner.`, following the pattern we already have on the `runner_nrunner.py` file, would make things easier to understand. This new parameter can be a copy of `runner.timeout.process_alive`. In the future, I think it will make things easy from the maintenance point of view.

Reply:
I was thinking about this too, but I'm not 100% convinced. For users, we should be transparent. Imagine that I do have some tests running on 'legacy runner" and I decide to change to the new runner with --test-runner='nrunner'. Forcing users to change all configuration options from 'runner.' to 'nrunner.' sounds to me a little bit unnecessary. And I understand that some configurations make sense only on a specific test-runner.

Today we have for instance 'runner.exectest.exitcodes.skip' that was introduced a few PRs ago and it is being used by the exec-test runner (runner).

But I agree with you, today it is confusing and we should choose one standard. I confess that I would like to unify this with "runner.".

Reply:
I agree with you that we should unify the options for the runners that are valid for all runners in one place.

My only concern here is the current feature name `runner.timeout.process_alive`. Can we rename that to something more meaningful for the `runner` and `nrunner`?

Reply:
What is your suggestion? Not sure if I understood. It starts with "runner." and has "timeout".

Reply:
Right, but then `process_alive` is really specific. If you look at the code from the `runner`, it talks explicitly about test processes. We can try to make it more generic. 

Thinking about the `nrunner`, this would be the `task` timeout, but we don't have a `task` definition on the `runner`.

So, to be honest, my best answer is I don't know, :D I'm not with a good imagination today

Reply:
`runner.timeout.process_alive` is used for *very* different purposes.  It's about the amount of time we'll wait after a *test* has said something like "I failed", and the process running that test actually terminates.  It's *not* about the amount of a time a test has to run and finish.

Now, I think we should not use *process* anywhere around the name of the configuration that controls the amount of time a nrunner Task is allowed to last.  We better use an abstract term for that.

Reply:
What about `task.timeout.running`?

Reply:
> What about `task.timeout.running`?

Sounds good to me.